static/app/views/monitors/edit.tsx

```diff
@@ -69,7 +72,21 @@ export default function EditMonitor() {
                   to: `/organizations/${organization.slug}/crons/`,
                 },
                 {
-                  label: t('Editing %s', monitor.name),
+                  label: (
+                    <MonitorBreadcrumb>
+                      <IdBadge
+                        project={monitor.project}
+                        avatarSize={16}
+                        hideName
+                        avatarProps={{hasTooltip: true, tooltip: monitor.project.slug}}
+                      />
+                      {monitor.name}
+                    </MonitorBreadcrumb>
+                  ),
+                  to: `/organizations/${organization.slug}/crons/${monitor.slug}/`,
```

Comment:
probably wont matter once we have the env selector in, but right now does it have the correct environment in the url?

Reply:
oh does it persist that

Reply:
yeah I see it 👍

Reply:
its okay tho lemme merge in my env selector and this should just work fine

Reply:
👍 cool